### PR TITLE
feat: refactored code

### PR DESCRIPTION
**Pull request by srv-pranjal**
# Change Logs

 - made changes suggested in reviews
 - added redirect to specific category on product page
 - updated favicon
 - added responsiveness to some components
 - fixed login not showing after page refresh
 - added document title on home page

### CODE REVIEW

File: src/Components/Category/Category.js [hunk 1]
```diff
@@ -5,6 +5,11 @@ import { CategoryCard } from "./CategoryCard";
 
 export const Category = () => {
   const [category, setCategory] = useState([]);
+
+  useEffect(() => {
+    document.title = "Home | Echo Store";
+  }, []);
```
Comment (viru2001) [inline src/Components/Category/Category.js]: you can make custom hook for changing document title

**Review by NamrataRaikwar2002 — APPROVED** [review]
Great work,

**Review by viru2001 — APPROVED** [review]
- good variable naming
- good folder structure
- Nice UI

Everything is good. you can merge this.

File: src/Components/Category/Category.js (hunk 1 above)
Comment (srv-pranjal) [reply src/Components/Category/Category.js]: done, Thanks

**Pull request merged**